### ISSUE TITLE
Sample code is having syntax error in TensorFlow nodejs code

### ISSUE THREAD

**Issue by varun-jalandery** [tensorflow/tensorflow#27361]
**System information**
- TensorFlow version:
- Doc Link:

**Syntax error in TensorFlow nodejs example**
[See sample code for Node.js usage](https://www.tensorflow.org/js/tutorials/setup)
Extra semicolon is causing the syntax error.

`onEpochEnd: (epoch, log) => console.log(`Epoch ${epoch}: loss = ${log.loss}`);`
In the above line, semi-colon in the end should be removed.
****

**Comment by ymodak** [comment]
The link you have referenced does not refer to the sample code you are referring too. Can you please take a look and provide pointers? Thanks!

**Comment by gadagashwini** [comment]
Sure Yasir


Thanks and Regards,
Ashwini Gadag

**Comment by ymodak** [comment]
Thanks for reporting. Closing since its  a duplicate #27237

**Comment by glaucopater** [comment]
Thanks for reporting @varun-jalandery. @ymodak I think you can close the original issue the #27237
